game/src/main/scala/io/aigar/game/Game.scala

```diff
@@ -1,32 +1,47 @@
 package io.aigar.game
 
 import io.aigar.game._
+import com.github.jpbetz.subspace._
 
 /**
  * Game holds the logic for an individual game being played
  * (e.g. the ranked game or a private test game).
  */
 object Game {
   final val RankedGameId = 0
+  final val PlayersInRankedGame = 15
```

Comment:
Need to discuss private games, but for ranked games I set it as a constant for now. Maybe we should revisit this once we consider settings auto-reload (having a file with all those balance settings)?

Also, supporting adding/removing players dynamically might be a tiny bit more complicated I believe, not sure we should do it here.